src/firebase.js

```diff
@@ -0,0 +1,15 @@
+import firebase from 'firebase/compat/app'
+import "firebase/auth"
+
+const app = firebase.initializeApp({
+  apiKey: "AIzaSyBaRLGZvIJY7YZuCHugptmfk0_2FosBU1M",
+  authDomain: "announce-d9b77.firebaseapp.com",
+  projectId: "announce-d9b77",
+  storageBucket: "announce-d9b77.appspot.com",
+  messagingSenderId: "252168177348",
+  appId: "1:252168177348:web:83d700cc72aabefb24eb91",
+  measurementId: "G-H0Q0Y49KQB"
+})
```

Comment:
For example, can you pull all these fields out to environment variables?

Reply:
I deleted to project and made a new one. I fixed it on my end, made a .env file with all the keys and stuff and updated firebase.js to get the keys from it. Is there a way to update the PR or should I close it and make a new one?